src/ImageBackingStore.cpp

```diff
@@ -1,8 +1,8 @@
 /**
- * ZuluSCSI™ - Copyright (c) 2022 Rabbit Hole Computing™
+ * Portions - Copyright (C) 2023 Eric Helgeson
+ * ZuluSCSI™ - Copyright (c) 2022-2023 Rabbit Hole Computing™
  *
  * This file is licensed under the GPL version 3 or any later version. 
- * It is derived from disk.c in SCSI2SD V6
```

Comment:
I agree, ImageBackingStore.cpp doesn't have SCSI2SD code in it, the notice has been accidentally copied over when it was separated from ZuluSCSI_disk.cpp.